MANUAL.md

```diff
@@ -0,0 +1,99 @@
+# Prometheus Genesis Kit Manual 
+The Prometheus Genesis Kit deploys a single-VM installation of
+Prometheus, Grafana, and AlertManager. It's capable of monitoring BOSH
+and Cloudfoundry.
+
+## Requirements
+
+[Node Exporter][1] must be installed on all BOSH VMs you would like
+to monitor. Please consult the README of that repository on how to
+setup the BOSH addon.
+
+
+## Features
+
+## SSL Certificates
+
+* `self-signed-certs` - If you wish to have Genesis generate
+  self-signed certs for you. 
+* `provided-cert` - If you have SSL cert/key to provide, which is
+  grabbed from Vault via path:
+  `$GENESIS_VAULT_PATH/nginx/ssl_certificate:certificate` and
+  `$GENESIS_VAULT_PATH/nginx/ssl_certificate:key`
+
+## Authentication
+
+* `http-auth` - By default, only Grafana supports authentication via
+  the web UI. This feature adds HTTP Basic Authentication to
+  AlertManager and Prometheus with automatically generated
+  credentials.
+
+## Monitoring
+
+* `monitor-cf` - Have Prometheus connect to the CF Firehose to track
+  CF app status + more. Requires two UAA accounts, one with the
+  `cloud_controller.admin_read_only` scope, and the `doppler.firehose`
+  scope. To create these accounts, here's an example:
+```
+uaac client add prometheus-cf \
```

Comment:
Regarding to two UAA clients, in our CF kit we have one UAA client `firehose` with both `cloud_controller.admin_read_only` and `doppler.firehose`authorities for this purpose. I like the idea of having two UAA clients in the PR, I also like the idea of our CF kit providing these two uaa clients, so we do not need run `uaac` command when we work with prometheus. Instead, in `genesis new` for prometheus,it will suggested the safe path for the two clients we already have in cf. 
What are your opinions? @jhunt @Proplex  @patrick1378

Reply:
I had planned on updating the `cf-genesis-kit` to automatically create the two necessary UAA accounts during a deployment, so that `prometheus-genesis-kit` could just grab them without asking. If the Prometheus kit saw that the credentials existed, it wouldn't prompt the operator during `genesis new`. Otherwise, if they were absent, it'd ask the operator for client id/secret.

Reply:
@Proplex That is even better! You got it all planned!

Reply:
😁Actually, I should add that functionality to the Prometheus kit now rather than cutting another release down the road. Will have that done tonight.

Reply:
I don't like the separate clients.  I prefer either using `firehose` (which is expressly for this), or creating a new `prometheus` client, if for some reason we don't like firehose.

Reply:
If we do create a new client, it should be an add-on, one that will generate and store the password in the correct location.